xrpl/core/binarycodec/types/serialized_dict.py

```diff
@@ -193,7 +195,18 @@ def from_value(
                 # keeps the original stack trace
                 e.args = (f"Error processing {field.name}: {e.args[0]}",) + e.args[1:]
                 raise
-            serializer.write_field_and_value(field, associated_value)
+            if field.name == "TransactionType" and str(associated_value) == "0066":
```

Comment:
nit: put `"0066"` into a final variable with a name describing it's a UNLModify transaction type.